app/controllers/mixins/ems_common_angular.rb

```diff
@@ -140,6 +140,8 @@ def get_task_args(ems)
         [user, params[:default_password], endpoint_opts]
       when 'ManageIQ::Providers::Lenovo::PhysicalInfraManager'
         [user, password, params[:default_hostname], params[:default_api_port], "token", false]
+      when 'ManageIQ::Providers::Hawkular::MiddlewareManager'
+        [user, password, params[:default_hostname], params[:default_api_port], params[:default_security_protocol], params[:default_tls_ca_certs]]
```

Comment:
In the provider we have `default_endpoint.ssl_cert_store` [1]

are these equivalent?

[1] https://github.com/ManageIQ/manageiq-providers-hawkular/blob/master/app/models/manageiq/providers/hawkular/middleware_manager.rb#L117

Reply:
I am gonna test with SSL but the field to insert the CA is default_tls_ca_certs

Reply:
I guess same question applies to  default_endpoint.security_protocol.
not sure how they are passed to middleware_manager and by who

Reply:
It seems that they are the fields we need
https://github.com/aljesusg/manageiq-ui-classic/blob/48bdaf4fae0523e1c5d5ddbce2ffa512e0c89621/app/controllers/mixins/ems_common_angular.rb#L183-L185

and i think this is where the connection request is sent:
https://github.com/aljesusg/manageiq-ui-classic/blob/48bdaf4fae0523e1c5d5ddbce2ffa512e0c89621/app/controllers/mixins/ems_common_angular.rb#L75

Reply:
Here is only getting the value of the field in the UI. And this is the Id of the field in the UI if you inspect code, I don't know when this value pass to default_endpoint.ssl_cert_store

Reply:
I agree with @aljesusg, the only data available in this call are taken from the angular form in the view. No provider data are available before the provider is created.